src/components/InviteLandingPage.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { useLocation, Navigate, useParams } from 'react-router-dom';
+import { PropTypes, instanceOf } from 'prop-types';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'PropTypes' is defined but never used.